MDSplus/mdsplus
# Feature: jTraverser and DeviceBeans

This is for sharing progress with @GabrieleManduchi 

TODO: 
- build jar and add to jTraverser2 startup scripts
- fix jTraverser.bat for windows
- add pydevice pkg
- update the javatraverser?.bat (issue #2021)
- maybe remove the .mak files

@GabrieleManduchi 
TEST new editors on dtacq example
- add pydevices from repo tp MDS_PYDEVICE_PATH
- start jTraverser2
- open new tree
- add ACQ2106_ACQ480x6
- check out Setup Device

TEST devicebeans support
- complie java/jdevices/src/jtraverser2/java into $OUTDIR
- $OUTDIR,jTraverser2.jar;jScope.jar;jDevices.jar to classpath and start jTraverser2
- open new tree
- add device
- check out Setup Device

waves and others seem to work except the ones mentioned in #1958
